OCR/frontend/src/components/FilePreview.test.tsx

```diff
@@ -1,136 +0,0 @@
-import { fireEvent, render, waitFor } from '@testing-library/react'
```

Comment:
why was this removed? is it not used?

Reply:
oh I see, it just got moved

Reply:
It has been moved into the tests folder.